src/zope/app/apidoc/utilities.rst

```diff
@@ -406,14 +408,16 @@ any positional arguments:
 
 Next we test whether the signature is correctly determined for class
 methods. Note that the `self` argument is removed from the signature, since it
-is not essential for documentation.
+is not essential for documentation (this only happens on Python 2 when
+we pass an "unbound method"; Python 3 doesn't have such a concept, so
+we always pass bound methods).
```

Comment:
I'm not sure I like this.

I'm not sure we always have bound methods to pass to API doc.  What if we can't construct an instance because the `__init__` needs arguments we don't know how to supply?